draft-ietf-quic-reliable-stream-reset.md

```diff
@@ -1,6 +1,5 @@
 ---
-title: "Reliable QUIC Stream Resets"
-abbrev: "QUIC Reliable Stream Reset"
+title: "QUIC Stream Resets with Partially Reliable Stream Data Delivery"
```

Comment:
```suggestion
title: "QUIC Stream Resets with Partially Reliable Delivery"
```

To me it seems "stream data" is verbose (what else would you deliver on streams)? Maybe "reliable" also is.